boards/arm/stm32h7/nucleo-h743zi/scripts/flash-mcu-loader.ld

```diff
@@ -0,0 +1,200 @@
+/****************************************************************************
```

Comment:
Please remove duplication. boards/arm/stm32h7/nucleo-h743zi/scripts/flash-mcuboot-loader.ld is already available